ui-framework-documentation.md

```diff
@@ -77,33 +90,83 @@ sub provide {
 ```
 
 main.pm then calls a scheduled [Test Module](#test-module) that is using
```

Comment:
If you want, you may also briefly describe, that while using 'yaml schedule' in main.pm we interrupt execution of main.pm [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/c41c425f6b72dc3cc437c9ad63af3e8c185fd05c/products/opensuse/main.pm#L128) and use [declarative scheduling](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/declarative-schedule-doc.md). So, that we do not rely on conditional logic in main.pm anymore.

Reply:
I think this explaination would better fir here https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/declarative-schedule-doc.md